ios/FastImage/FFFastImageView.m

```diff
@@ -142,12 +144,21 @@ - (void)reloadImage
         }
         
         // Set headers.
-        [_source.headers enumerateKeysAndObjectsUsingBlock:^(NSString *key, NSString* header, BOOL *stop) {
-            [[SDWebImageDownloader sharedDownloader] setValue:header forHTTPHeaderField:key];
+        NSDictionary *headers = _source.headers;
+        SDWebImageDownloaderRequestModifier *requestModifier = [SDWebImageDownloaderRequestModifier requestModifierWithBlock:^NSURLRequest * _Nullable(NSURLRequest * _Nonnull request) {
+            NSMutableURLRequest *mutableRequest = [request mutableCopy];
+            for (NSString *header in headers) {
+                NSString *value = headers[header];
+                [mutableRequest setValue:value forHTTPHeaderField:header];
+            }
+            return [mutableRequest copy];
         }];
+        SDWebImageContext *context = @{SDWebImageContextDownloadRequestModifier : requestModifier};
         
         // Set priority.
         SDWebImageOptions options = SDWebImageRetryFailed | SDWebImageHandleCookies;
+        // Bypass invalid SSL certificate, same behavior like React Native
+        options |= SDWebImageAllowInvalidSSLCertificates;
```

Comment:
I wonder about this. This lib allows specifying HTTP headers, which would most likely be used for authentication. This option would allow a man-in-the-middle attacker to collect headers.

Should this be under a flag, as a separate PR/issue? 

I feel more thought is needed here. In my case, the changes in the PR work without this flag too.

Reply:
I think react-native-fast-image should have a config via JS API for this. Current they have no such of thing.

The React Native already allows the un-trusted SSL error. I just do the same behavior to match them.

If you really don't want this, I can just revert.

Reply:
Try again. Maybe now some bad image server may not load the image. (However, React Native's `<Image>` can )

Reply:
Do you have a link pointing out to this behavior in the RN Image component? I couldn't find anything.

Reply:
Seems I misunderstand the behavior. RN does not add the delegate to `URLSession:task:didReceiveChallenge:completionHandler:`. When self-host or some SSL error happaned, it will use iOS's default behavior.

However, this default behavior seems different from SDWebimage's code (which provide a default behavior when `SDWebImageAllowInvalidSSLCertificates` not passed).

Current keep this.